package.json

```diff
@@ -45,18 +45,18 @@
     "release": "standard-version"
   },
   "dependencies": {
-    "@energyweb/credential-governance": "^2.2.1-alpha.291.0",
-    "@energyweb/vc-verification": "^2.2.1-alpha.291.0",
+    "@energyweb/credential-governance": "2.2.1-alpha.303.0",
+    "@energyweb/vc-verification": "2.2.1-alpha.303.0",
     "@ensdomains/ens": "^0.4.5",
     "@ensdomains/resolver": "^0.2.4",
-    "@ew-did-registry/claims": "^0.7.1-alpha.795.0",
-    "@ew-did-registry/credentials-interface": "^0.7.1-alpha.795.0",
-    "@ew-did-registry/did-document": "^0.7.1-alpha.795.0",
-    "@ew-did-registry/did-ethr-resolver": "^0.7.1-alpha.795.0",
-    "@ew-did-registry/did-ipfs-store": "^0.7.1-alpha.795.0",
-    "@ew-did-registry/jwt": "^0.7.1-alpha.795.0",
-    "@ew-did-registry/keys": "^0.7.1-alpha.795.0",
-    "@ew-did-registry/proxyidentity": "^0.7.1-alpha.795.0",
+    "@ew-did-registry/claims": "0.8.1-alpha.933.0",
+    "@ew-did-registry/credentials-interface": "0.8.1-alpha.933.0",
+    "@ew-did-registry/did-document": "0.8.1-alpha.933.0",
```

Comment:
@JGiter I see you already have an open PR to upgrade `ew-did-registry` to v8.  Do you think we can upgrade that here ? Or should wait for the IPFS changes to be finalised?

Reply:
What version should I update to if not? This morning in the meeting we said if I used the latest Of ew-credentials, I should use the latest of ew-did-registry as well.

Reply:
did-reg v0.8 has breaking changes. We should wait for merging PR integrates these changes in ssi-hub